LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaPayloadDecoder.cs

```diff
@@ -131,13 +132,17 @@ async Task<string> CallSensorDecoderModule(string sensorDecoderModuleUrl, byte[]
         /// <summary>
         /// Value sensor decoding, from <see cref="byte[]"/> to <see cref="string"/>
         /// </summary>
-        /// <param name="payload"></param>
-        /// <param name="fport"></param>
-        /// <returns></returns>
+        /// <param name="payload">The payload to decode</param>
+        /// <param name="fport">The received frame port</param>
+        /// <returns>The decoded value as a JSON string</returns>
         public static string DecoderValueSensor(byte[] payload, uint fport)
         {
             var result = Encoding.UTF8.GetString(payload);
-            return $"{{ \"value\":{result} }}";
+
+            var isSupportedNumericValue = double.TryParse(result, NumberStyles.Float, CultureInfo.InvariantCulture, out double ignored);
```

Comment:
Out of curiosity: why are you specifying a number style?

Reply:
To be specific about what characters are supported in the JSON spec. If you have en-US culture and you pass in 1,643.57 without specifying a NumberStyle it will parse fine, but the JSON parser would throw.